app/controllers/posts_controller.rb

```diff
@@ -1,8 +1,30 @@
 # frozen_string_literal: true
 
 class PostsController < ApplicationController
+  before_action :load_post!, only: %i[show]
+
   def index
     posts = Post.all
     render status: :ok, json: { posts: }
   end
+
+  def create
+    post = Post.new(post_params)
+    post.save!
```

Comment:
```suggestion
    Post.create!(post_params)
```
This is would do, it is much cleaner.